### PR TITLE
[Improvement] Modify some configuration's names and default values

### PR DESCRIPTION
**Pull request by jerqi**
### What changes were proposed in this pull request?
Modify some configuration's names and default values.

### Why are the changes needed?
Unify configuration option style, and update proper default value for configuration.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
GA passed

### CODE REVIEW

File: server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java [hunk 1]
```diff
@@ -123,7 +123,7 @@
   public static final ConfigOption<Long> SERVER_PRE_ALLOCATION_EXPIRED = ConfigOptions
       .key("rss.server.preAllocation.expired")
       .longType()
-      .defaultValue(10 * 1000L)
+      .defaultValue(120 * 1000L)
```
Comment (colinmjj) [inline server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java:126]: 120 is too long for preAllocation expired

Comment (jerqi) [reply server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java:126]: In sop and online ,  we choose 120s. What's your suggestion?

Comment (colinmjj) [reply server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java:126]: when client send data to shuffle server, request memory first, then send data if there has enough memory. Shuffle server will keep the preAllocation memory until preAllocation.expired. If application is killed, there will be a lot of preAllocation memory in shuffle server.
preAllocation.expired is about interval of 2 rpc calls. If 10s is too short, it means there has some problem in rpc communication, maybe overload, etc.  These problem should be resolved first, preAllocation.expired is just a workaround for these problem.
I suggest to set 20s as default value at most

File: common/src/main/java/com/tencent/rss/common/config/RssBaseConf.java [hunk 2]
```diff
@@ -110,7 +110,7 @@
   public static final ConfigOption<Integer> RPC_MESSAGE_MAX_SIZE = ConfigOptions
       .key("rss.rpc.message.max.size")
       .intType()
-      .defaultValue(Integer.MAX_VALUE)
+      .defaultValue(1073741824)
```
Comment (duanmeng) [inline common/src/main/java/com/tencent/rss/common/config/RssBaseConf.java:113]: 1073741824 -> 1024 * 1024  * 1024 to make it readable.

File: server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java (hunk 1 above)
Comment (jerqi) [reply server/src/main/java/com/tencent/rss/server/ShuffleServerConf.java:126]: OK

File: common/src/main/java/com/tencent/rss/common/config/RssBaseConf.java (hunk 2 above)
Comment (jerqi) [reply common/src/main/java/com/tencent/rss/common/config/RssBaseConf.java:113]: OK

**Comment by jerqi** [comment]
@duanmeng @colinmjj all comments are addressed.

**Comment by jerqi** [comment]
@duanmeng @colinmjj gently ping.

**Review by colinmjj — APPROVED** [review]
LGTM

**Pull request merged**